UofS-Pulse-Binfo/analyzedphenotypes
# Update download and trait page

Update to AP data downloader to support new system variables and improved germplasm name selection as filter option.

How to test and other details available in this PR:
https://github.com/UofS-Pulse-Binfo/analyzedphenotypes/pull/7

Please note that this PR requires Data Upload PR and Module Configuration PR (to ensure all apis are in).
